### PR TITLE
Prow testing

### PR DESCRIPTION
**Pull request by pohly**
This imports csi-release-tools (unchanged copy) and adds a .prow.sh file in 944be88.

### CODE REVIEW

**Comment by pohly** [comment]
/hold

https://github.com/kubernetes/test-infra/pull/12088 has to be merged first

File: release-tools/util.sh [hunk 1]
```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/env bash
+
+# Copyright 2014 The Kubernetes Authors.
```
Comment (xing-yang) [inline release-tools/util.sh]: nit: should be 2019

File: release-tools/verify-shellcheck.sh [hunk 2]
```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env bash
+
+# Copyright 2018 The Kubernetes Authors.
```
Comment (xing-yang) [inline release-tools/verify-shellcheck.sh]: 2019

File: release-tools/util.sh (hunk 1 above)
Comment (pohly) [reply release-tools/util.sh]: The original code actually is from 2014. I just copied it.

File: release-tools/verify-shellcheck.sh (hunk 2 above)
Comment (pohly) [reply release-tools/verify-shellcheck.sh]: Same here.

**Comment by pohly** [comment]
@xing-yang I can't change these commits here. They are verbatim copies from https://github.com/kubernetes-csi/csi-release-tools/commits/master

If you want to review such changes before they are merged, then I can add you as reviewer of future changes in that repo.

If you see something that absolutely has to be changed now, then we'll have to treat that as bugs against csi-release-tools. But these nits are probably something that we can leave unchanged. Agreed?

**Comment by xing-yang** [comment]
@pohly If these are copies from https://github.com/kubernetes-csi/csi-release-tools/commits/master and cannot be changed in this repo, can you add some description at the top of this PR to clarify that?

**Comment by pohly** [comment]
> If these are copies from https://github.com/kubernetes-csi/csi-release-tools/commits/master and cannot be changed in this repo, can you add some description at the top of this PR to clarify that?

I was assuming that component maintainers are familiar with how `csi-release-tools` (= `release-tools` in each component) is maintained. But sure, I can add something.

**Pull request merged**